src/main/java/Parser.java

```diff
@@ -0,0 +1,6 @@
+/**
+ * Parser deals with making sense of the user command.
+ */
+public class Parser {
```

Comment:
Perhaps Parser could have been completed before pushing to github? If not, some reviewers might be confused at the empty class implementation.